client/styles/components/_account.scss

```diff
@@ -29,3 +29,22 @@
 .account__social-stack > * {
   margin-right: #{15 / $base-font-size}rem;
 }
+
+// @media screen and (max-width: 767px) {
+//   .social_button_github,
+//   .social_button_google {
+//     display: block;
+//     width: 100%;
+//     margin-bottom: 10px;
+//   }
+// }
+
+// @media screen and (min-width: 10%) {
+//   .social_button_github,
+//   .social_button_google {
+//     display: inline-block;
+//     width: auto;
+//     margin-bottom: 0;
+//   }
+// }
+
```

Comment:
It's probably better to delete this than to add commented-out styles into the codebase.